.github/taskfiles/docs.yml

```
@@ -0,0 +1,148 @@
+      - mkdir -p _book
+      - touch _book/links.json
+      - |
+        printf "${BLUE}[INFO] Copy API docs...${RESET}\n"
+        if [ -d _pdoc ]; then
+          mkdir -p _book/pdoc
+          cp -r _pdoc/* _book/pdoc
+          echo '{"API": "./pdoc/index.html"}' > _book/links.json
+        else
+          echo '{}' > _book/links.json
+        fi
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)